### PR TITLE
DOCSP-41417-add-db2-support-content

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

- Adding content to multiple pages to document Relational Migrator support for DB2.

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-41417/installation/install-on-a-local-machine/install-ubuntu/ (Applies to all Install pages)

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-41417/installation/install-on-a-local-machine/install-on-a-local-machine/#std-label-local-machine-install

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-41417/jobs/prerequisites/db2/

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-41417/connection-strings/relational-database-connection-strings/#db2

## JIRA

https://jira.mongodb.org/browse/DOCSP-41417

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66bf9a038e811ae6727ef86c

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/projects/create-project-loading-schema-files.txt [hunk 1]
```diff
@@ -108,6 +108,11 @@ to generate DDL files from your relational system.
 
       Importing DDL files with Sybase ASE is not supported.
 
+   .. tab:: DB2
+      :tabid: db2-ddl
+
+      Importing DDL files with DB2 is not supported.
```
Comment (ianf-mongodb) [inline source/projects/create-project-loading-schema-files.txt:114]: Can you confirm @tomhollander or @alwaysbemark?

File: source/jobs/prerequisites/db2.txt [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+- Relational Migrator does not create any indexes on your DB2 instance to 
+  facilitate sync jobs. The create index creation permission is not 
+  required.
+
+- Creating the required schema and table have no impact on database performance.
+
+- Relational Migrator does not support ``BLOB``, ``CBLOB`` or ``DBCLOB`` columns 
+  of DB2 tables. If this feature is important to you, create or upvote the 
+  feature request on `feedback.mongodb.com <https://feedback.mongodb.com/forums/952450-relational-migrator>`__.
```
Comment (ianf-mongodb) [inline source/jobs/prerequisites/db2.txt:45]: Added this let me know if you dont want to include in the docs @tomhollander

File: source/jobs/creating-jobs.txt [hunk 3]
```diff
@@ -75,7 +75,9 @@ Steps
           - Leaving database name blank loads schemas from the default database.
         * - Sybase ASE
           - Leaving database name blank loads schemas from the default database.
-  
+        * - DB2
+          - Leaving database name blank loads schemas from the default database. (need confirmation)
```
Comment (ianf-mongodb) [inline source/jobs/creating-jobs.txt:79]: Need confirmation on this also @alwaysbemark  @tomhollander

**Comment by ianf-mongodb** [comment]
@tomhollander @alwaysbemark this is ready for inital review, could you have a look please.

File: source/jobs/creating-jobs.txt (hunk 3 above)
Comment (alwaysbemark) [reply source/jobs/creating-jobs.txt:79]: They have to specify a database, not pointing to one will lead to a syntax error on the UI

File: source/projects/create-project-loading-schema-files.txt (hunk 1 above)
Comment (alwaysbemark) [reply source/projects/create-project-loading-schema-files.txt:114]: Confirming, not supported!

File: source/jobs/prerequisites/db2.txt [hunk 4]
```diff
@@ -0,0 +1,73 @@
+.. _rm-prereq-db2:
```
Comment (alwaysbemark) [inline source/jobs/prerequisites/db2.txt:1]: Thanks for writing this up Ian!

**Review by alwaysbemark — APPROVED** [review]
Looks good to me, I've added the necessary confirmations and clarifications. Thanks Ian!

**Review by tomhollander — APPROVED** [review]

File: source/jobs/prerequisites/db2.txt [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+Configure Migration Prerequisites for DB2
+=========================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+To run sync jobs from a DB2 source database, the database requires
```
Comment (mmaville-mdb) [inline source/jobs/prerequisites/db2.txt:13]: Minor: I might suggest adding target or some other identifying information for the second database mentioned in the sentence.

```suggestion
To run sync jobs from a DB2 source database, the target database requires
```

**Review by mmaville-mdb — APPROVED** [review]
LGTM! One minor suggestion.

File: source/installation/install-on-a-local-machine/install-on-a-local-machine.txt [hunk 6]
```diff
@@ -44,6 +44,10 @@ If you are migrating from an Oracle database:
 - (Optional) To migrate tables that contain the ``XMLType`` type, you must also download the 
   `xdb.jar <https://www.oracle.com/database/technologies/appdev/jdbc-drivers-archive.html>`__ driver. 
 
+If you are migrating from an DB2 database:
```
Comment (mmaville-mdb) [inline source/installation/install-on-a-local-machine/install-on-a-local-machine.txt:47]:
```suggestion
If you are migrating from a DB2 database:
```

**Pull request merged**